src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
The class name `ReturnOperationHandler` does not follow the Java naming conventions. Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to simplify, but do not use abbreviations.